openconcept/analysis/openaerostruct/aerostructural.py

```diff
@@ -60,8 +59,8 @@ class OASAerostructDragPolar(om.Group):
     This component cannot currently handle fuel loads on the wing,
     nor can it handle point loads applied to the structure.
 
-    NOTE: the spanwise variables (twist, toverc, skin/spar thickness) are ordered
-          starting at the tip and moving to the root; a twist of [-1, 0, 1] would
+    NOTE: the spanwise variables (twist, toverc, skin/spar thickness) are ordered \
```

Comment:
Why backslashes?

Reply:
It was messing up the notes in the build (the first line sometimes would get highlighted in grey as a note and the following lines ended up as plain text). Is there a better way that looks good in the code and the docs?

Reply:
These can either go under the description (i.e. before any sections), or the [notes section](https://numpydoc.readthedocs.io/en/stable/format.html#notes).